.eslintrc.json

```diff
@@ -74,6 +74,13 @@
 		"no-negated-in-lhs": "error",
 		"no-nested-ternary": "error",
 		"no-redeclare": "error",
+		"no-restricted-syntax": [
+			"error",
+			{
+				"selector": "ImportDeclaration[source.value=/^@wordpress\\u002F.+\\u002F/]",
```

Comment:
The need to traverse into path structure of a dependency is few and far between. I'd be more concerned about all of the places we have [this pattern](https://github.com/WordPress/gutenberg/blob/d95faa5/package.json#L104) repeated throughout the codebase, particularly if/when we add/remove from the list.